back/db/migrate/20220707102050_add_uniqueness_constraint_to_volunteers.volunteering.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This migration comes from volunteering (originally 20220707091502)
+
+class AddUniquenessConstraintToVolunteers < ActiveRecord::Migration[6.1]
+  def change
+    ActiveRecord::Base.connection.execute(
+      # After https://stackoverflow.com/a/12963112/3585671
+      <<-SQL.squish
+      DELETE FROM volunteering_volunteers to_delete USING (
```

Comment:
Magic :sparkles: for me :grimacing: .
If you have verified locally that one of the duplicates is kept let's go with it.